packages/lisk-core/src/channels/event_emitter.js

```diff
@@ -0,0 +1,62 @@
+const Event = require('../event');
+const Action = require('../action');
+const BaseChannel = require('./base');
+
+module.exports = class EventEmitterChannel extends BaseChannel {
+	constructor(moduleAlias, events, actions, bus, options = {}) {
```

Comment:
It's unclear to me, why are we passing bus instance here but not in the other Channels.